### PR TITLE
RSDK-4734 More polishing of CLI

### PR DESCRIPTION
**Pull request by benjirewis**
RSDK-4734

Avoids base URL logic for logging out (just clears the cache). Adds a new `printf` method to replace the numerous `fmt.Fprintf` calls. Capitalizes all printed messages and errors returned to the user (Go standard is lowercase but _CLI standard_ is uppercase; maintains all errors as lowercase-starting internally). Modifies tests accordingly.

### CODE REVIEW

File: .gitignore
Comment (benjirewis) [inline .gitignore:45]: One of the CLI tests creates this data.

File: cli/auth.go
Comment (zaporter-work) [inline cli/auth.go:117]: :joy: 

Might just be better to duplicate the printf. Fine as it is though.

**Review by zaporter-work — APPROVED** [review]
LGTM!

I don't spot any typos.

File: cli/auth.go
Comment (ohEmily) [reply cli/auth.go:117]: 👏 

[ignorable] fwiw another approach would be to capitalize the first letter inside the printf function (could control it with a boolean if you want)

**Review by ohEmily — APPROVED** [review]
Beautiful! Thanks for taking the time to do this. I believe that these changes will improve customer confidence in the product's finish once CLI usage is more widespread.

File: cli/auth.go
Comment (benjirewis) [reply cli/auth.go:117]: Thanks for suggestions! I think I'll just leave as-is for now; the auto-capitalizing logic is kind of confusing and will make it hard to grep for messages in the CLI. I don't even like having it for errors, but 🤷🏻 . I think I'd rather devs write messages here and have them printed as close to what they're writing as possible.

**Comment by benjirewis** [comment]
Will wait for #2856 to merge before rebasing and merging this.

**Pull request merged**
